readabilitySAX.js

```diff
@@ -206,7 +206,10 @@ Readability.prototype._settings = {
 	linksToSkip: {},	//pages that are already parsed
 	//pageURL: null,	//URL of the page which is parsed
 	//type: "html",		//default type of output
-	resolvePaths: false
+	resolvePaths: false,
+    includeEmbeds: function(src) {
```

Comment:
Default method implementation does not override the existing embed handling. This setting is provided to support, if user wants to allow a different set of embeds than supported by re_videos regular expression. Or did you mean something else ?